decidim-assemblies/app/controllers/decidim/assemblies/admin/assembly_publications_controller.rb

```diff
@@ -27,7 +27,7 @@ def create
         def destroy
           authorize! :publish, current_assembly
 
-          UnpublishAssembly.call(current_assembly) do
+          UnpublishAssembly.call(current_assembly, current_user) do
```

Comment:
Food for thought: Maybe we should do with commands the same we've done with form: Provide a `command(symbol)` helper that populates the contextual fields by default.

Reply:
But you don't know if you need the form for a given command or not